pkg/compose/verify.Dockerfile

```diff
@@ -53,6 +54,7 @@ FROM ${PKG_BASE_IMAGE} AS verify-rpm
 COPY --from=xx / /
 ARG PKG_RELEASE
 ARG PKG_DISTRO
+ARG PKG_ID
```

Comment:
Wondering if `PKG_DISTRO_ID` (to indicate these are related to the distro) would be clearer (I guess same for `PKG_DISTRO_SUITE`)

It's a bit longer of course 🤔

Reply:
Sure, should be good.